tests/browser_context_test.go

```diff
@@ -0,0 +1,82 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/dop251/goja"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBrowserContextAddCookies(t *testing.T) {
+	opts := defaultLaunchOpts()
+	tb := newTestBrowser(t, withFileServer(), opts)
+
+	t.Run("happy_path", func(t *testing.T) {
+		testCookieName := "test_cookie_name"
+		testCookieValue := "test_cookie_value"
+
+		bc := tb.NewContext(nil)
+		cmd := fmt.Sprintf(`
+			[
+				{
+					name: "%v",
+					value: "%v",
+					url: "%v"
+				}
+			];
+		`, testCookieName, testCookieValue, tb.URL(""))
+		cookies, err := tb.vu.Runtime().RunString(cmd)
```

Comment:
Could you use the helper function [`testBrowser.runJavaScript`](https://github.com/grafana/xk6-browser/blob/main/tests/test_browser.go#L232) here?